package.json

```diff
@@ -41,7 +42,8 @@
     "redux-persist": "^6.0.0",
     "redux-thunk": "^2.3.0",
     "request": "^2.88.2",
-    "serverless-http": "^2.7.0"
+    "serverless-http": "^2.7.0",
+    "styled-components": "^5.3.3"
```

Comment:
Why do you add fancy components? Can you use CSS classes defined in files? It is to follow the standard in the project and not add more handles of different styles.

Reply:
please remove it. The idea is to keep the current implementation and not add more variations in the handling of styles in the project